playbook/app/pb_kits/playbook/pb_section_separator/docs/_section_separator_vertical.html.erb

```diff
@@ -11,3 +13,41 @@
         Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
     <% end %>
 <% end %>
+</br>
+<div class="clear"></div>
+<%= pb_rails("title", props: { text: "Flex Container", tag: "h4", size: 4 }) %> </br>
+  <%= pb_rails("flex", props: {orientation: "column", inline:false, classname: "flexSSeperatortext"}) do %>
+    <%= pb_rails("flex") do %>    
+        <%= pb_rails("section_separator", props: { orientation: "vertical", classname: "flexSSeperatorContainer" }) %> 
+        <%= pb_rails("body", props: { classname: "flexSSeperatortext" }) do %>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
+        <% end %>
+        <% end %>
+  <% end %>
+
+<div class="clear"></div>
+</br>
+<%= pb_rails("title", props: { text: "Fixed Height", tag: "h4", size: 4 }) %> </br>
+<%= pb_rails("section_separator", props: { orientation: "vertical", classname: "fixedHeightSSeparator" }) %>
+<%= pb_rails("body", props: { classname: "SFixedHeight" }) do %>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua
+<% end %>
+<div class="clear"></div>
+</br>
+<%= pb_rails("title", props: { text: "Auto Height", tag: "h4", size: 4 }) %> </br>
```

Comment:
Title kit will use `h3` tag by default. Technically tag is defined in relation to its role in the document. Here you want to have a `span` tag unless h4 is required here.